scripts/skydb.js

```diff
@@ -11,8 +11,9 @@ const fs = require("fs");
 const process = require("process");
 
 (async () => {
-  const { SkynetClient, genKeyPairFromSeed, formatSkylink } = require("..");
-  const client = new SkynetClient("https://siasky.net");
+  const { SkynetClient, defaultSkynetPortalUrl, genKeyPairFromSeed, formatSkylink } = require("..");
+  const portalUrl = defaultSkynetPortalUrl;
+  const client = new SkynetClient(`${portalUrl}`);
```

Comment:
Hello kwypchlo,

I would make the changes but I'm not sure if I should do it as m-cat made the last change to the PR.
If I now push my outdated PR over it, I don't know whether the changes from m-cat will still be there.
I don't think so, right?

But I also don't show how I can update my PR, so that the changes are available from m-cat ( 5b3fc1d ).

Reply:
You should run `git pull` locally to pull latest changes on your branch. Let me know if that helped, otherwise we can skip my comment since it's not like it's wrong, it's just redundant.